### PR TITLE
Bookstore: styling 🦖 

### PR DESCRIPTION
**Pull request by ismailmunyentwari9**
# WHAT I HAVE DONE IN THIS PART OF PROJECT 🌹 
- [ ] Add  missing JSX elements to  project.
- [ ] Your final design match Zeplin's design .
- [ ] Use the CSS rules detailed in Zeplin to style the website.
# 🧠  SKILLS i GAINED IN THIS PART OF PROJECT 🌳 

- [ ] Understand the concept of state management.
- [ ] Use store, actions and reducers in React.
- [ ] Connect React and Redux.
- [ ] Apply React best practices and language style guides in code.

### CODE REVIEW

File: src/components/book.jsx [hunk 1]
```diff
@@ -24,7 +15,7 @@ function Book({ bookstore }) {
               <div className="action-button">
                 <button type="button">Comments</button>
                 <span className="vertical-divider" />
-                <button type="button" onClick={() => handleDelete(items.id)}>Remove</button>
+                <button type="button" onClick={() => handleRemoveBook(items.item_id)}>Remove</button>
                 <span className="vertical-divider" />
                 <button type="button">Edit</button>
                 <span className="vertical-divider" />
```
Comment (Munsa1) [inline src/components/book.jsx]: ## Due to inssuficient files changed I will leave comment here
- [ ] In `book.css` You have repeating CSS rules. Kindly create a reusble classes for `width` which occurs in line 53 and 67. Also create a reusable class for `display: flex` which is used multiple times as well

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @ismailmunyentwari9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
### Due to insufficient files changed I will a comment here
- [ ] Kindly fix the broken link in the `Getting Started` section of your ReadMe file
image
- [ ] Also since you have not deployed the application, kindly remove the `Live Demo` section from your ReadMe file
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @ismailmunyentwari9 🙋‍♂️
Good job working on implementing the changes required by the previous reviewer. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is not yet professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- [ ] As the previous reviewer suggest, you should ensure to follow the [best CSS practices](https://kinsta.com/blog/css-best-practices/) which one of them includes using the reusable piece of code to avoid repeatition. Therefore i suggest that you apply the changes required about avoiding the `display:flex` and `width` at line 53 and line 63.

## Optional suggestions
- [ ] [OPTIONAL] As the previous reviewer suggested, Kindly ensure that the apprearing link is not broken. If you notice that is not supposed to be a link but instead a placeholder. Kindly ensure to fix that to have a professional README file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by emmiiorji — APPROVED** [review]
Hi @ismailmunyentwari9, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>


## Good Highlights✔️

- Design follows the template quite well
- Requested changes have been implemented

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**